### DISCUSSION TITLE
Setting up city level data, fresh install only shows countries

### DISCUSSION THREAD

**Discussion by ConnorOnTheWeb** [plausible/analytics#1601] [Self-Hosted Support] [answered]
I've got Plausible self-hosted running and I'm loving it.

Only problem is I can only see country-level data, not city-level.

![Screenshot_20220110_210538](https://user-images.githubusercontent.com/28314904/148993486-4aae0082-7d3f-400e-b254-a9685ca24493.png)

Fresh install on Ubuntu 20.04. Only saw countries, so I looked at this doc ([plausible.io/docs/self-hosting](https://plausible.io/docs/self-hosting#1-maxmind-geolocation-database)) and went through those steps.

Still not seeing city level, my view remains the same as the above screenshot.

Is there something else I'm not doing?

**Comment by metmarkosaric** [comment, marked as answer]
Cities is a brand new feature on Plausible and available only on the cloud version for now. Self-hosted version is not updated as frequently as the hosted version which changes daily.

Think of it as a long term release to help us and help you not need to upgrade things all the time and risk breaking stuff. We stack all the new features and fixes that have been battle tested on the hosted version and release a new self-hosted version twice a year. 

You can learn more here: https://plausible.io/blog/building-open-source

**Reply by ConnorOnTheWeb** [reply]
Got it, thank you for clarifying this distinction.

When cities is available for self-hosted, will the analytics I'm collecting now be viewable by city?

**Reply by metmarkosaric** [reply]
No as we've changed the database in order to have accurate city data and this database we're paying to have a license for. Too early to say how this will work in the self-hosted version. We could use the old database for self-hosted even though it's less accurate as otherwise every self-hoster would need to pay for the same license that we're paying for. Still early so there will be more info as we get closer to the new self-hosted release this summer.

**Reply by ConnorOnTheWeb** [reply]
Thanks again for letting me know how it goes. Signed up for Plausible cloud today, I'm sold after the past few days of self-hosted. Proud to support this project, thank you!

**Reply by metmarkosaric** [reply]
glad to hear that @ConnorOnTheWeb, enjoy Plausible!

**Comment by wheeleran** [comment]
any update on this a year and a half later, Cities with self-hosted?

**Reply by ruslandoga** [reply]
@wheeleran 👋

Yes! Please see https://github.com/plausible/analytics/discussions/2507

**Comment by ruslandoga** [comment]
👋

To anyone who finds this issue: in the new (pre-)release https://github.com/plausible/analytics/discussions/2896 city-level geolocation is enabled automatically.
